node/toolrunner.ts

```diff
@@ -82,10 +84,10 @@ export class ToolRunner extends events.EventEmitter {
         var args = [];
 
         var inQuotes = false;
-        var escaped =false;
+        var escaped = false;
```

Comment:
@ericsciple: I ran format command in VSCode that brought in other changes. If you don't want these committed, I can back them out .